Tasks/UniversalPackagesV0/universalpublish.ts

```diff
@@ -149,6 +149,12 @@ export async function run(artifactToolPath: string): Promise<void> {
             retries++; 
         }
 
+        if(publishStatus === packageAlreadyExistsError){
```

Comment:
Thanks for catching the bug. This if check can be moved to line 118 before the while loop